dlp/deid.py

```diff
@@ -83,8 +83,72 @@ def deidentify_with_mask(
 
 # [END dlp_deidentify_masking]
 
+# [START dlp_deidentify_replace]
+def deidentify_with_replace(
+    project,
+    string,
+    info_types,
+    replacement_str=None,
+):
+    """Uses the Data Loss Prevention API to deidentify sensitive data in a
+    string by replacing matched input values with a value you specify.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        string: The string to deidentify (will be treated as text).
```

Comment:
`info_types` is missing in docstring

Reply:
Again, I was trying to follow surrounding code for consistency, which also didn't have info_types in the docstring. In python, I think it's ok to leave out a parameter from the docstring if their meaning is clear enough.

I can create a separate pull request for improving the entire python style for this repo.

Reply:
I think `info_types` missing was a mistake. See https://github.com/GoogleCloudPlatform/python-docs-samples/blob/4b0d97e95cf5b88387671187deb5d9e95ede7fa7/dlp/inspect_content.py#L40 for comparison. Would you mind updating it for the two functions in this file?

Reply:
You are right. It should be present for all functions. This pull request only concerns with deid.py. I'll create a separate pull request for docstring changes for the entire repo. Would that be ok or do you suggest making these nits here?

Reply:
I don't have a strong preference either way, please do whatever is easier for you

Reply:
Added info_types to the new changes. Will create a separate request for other files.